samples/apps/json.jakt

```diff
@@ -223,37 +245,95 @@ class JsonParser {
         return JsonValue::Object(values)
     }
 
+    function char_to_f64(anon num: u8) throws -> f64 {
+        // FIXME 1: Shouldn't need this function at all
```

Comment:
The first fixme can be put before the function definition, the second one can be put at the return statement that shouldn't be necssary